clustered/common/src/main/java/org/ehcache/clustered/common/ServerStoreCompatibility.java

```diff
@@ -76,6 +77,26 @@ public void verify(ServerStoreConfiguration serverConfiguration, ServerStoreConf
     if (!isCompatible) {
       throw new ClusteredStoreValidationException(sb.toString());
     }
+
+  }
+
+  private static boolean comparePoolAllocationType(StringBuilder sb, PoolAllocation serverPoolAllocation, PoolAllocation clientPoolAllocation) {
+    if ((serverPoolAllocation == null && clientPoolAllocation == null)
```

Comment:
I removed the null checks.  It appears that this case can't happen.  If I am wrong please correct me.